bumpversion/cli.py

```diff
@@ -619,29 +657,29 @@ def main(original_args=None):
 
         if do_commit:
             vcs.add_path(path)
-
     vcs_context = {
         "current_version": args.current_version,
         "new_version": args.new_version,
     }
     vcs_context.update(time_context)
```

Comment:
This and the next 2 lines make me wonder if it should use `context` that's output of `_assemble_context`.

Reply:
I'll give it a try.